LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2017 Target Brands, Inc.
```

Comment:
This is actually invalid - nobody should ever modify the license. It's provided as-is, and contributors or any such things are placed in the `NOTICE` files

Reply:
@timperrett I'm curious, do you have a source for this? This seems explicitly designed to be filled in, and plenty of projects (including e.g. Apache OpenOffice) do fill it in.

Reply:
@shlevy actually I don't - this was instruction from our legal group. Having just combed the license, I notice this: 

```
   APPENDIX: How to apply the Apache License to your work.

      To apply the Apache License to your work, attach the following
      boilerplate notice, with the fields enclosed by brackets "[]"
      replaced with your own identifying information. (Don't include
      the brackets!)  The text should be enclosed in the appropriate
      comment syntax for the file format. We also recommend that a
      file or class name and description of purpose be included on the
      same "printed page" as the copyright notice for easier
      identification within third-party archives.

   Copyright [yyyy] [name of copyright owner].
```

Which does indeed look like it should be filled out, and is otherwise exempt from the "Your modifications" sub-section.